tests/ui-toml/disallowed_profiles_methods/main.rs

```diff
@@ -12,6 +12,12 @@ fn default_violation() {
     std::mem::drop(value); //~ ERROR: use of a disallowed method `std::mem::drop`
 }
 
+#[expect(clippy::disallowed_methods)]
```

Comment:
We also need expects for the `unknown_profile` lint, and trying it before and after the `profile` attribute.